bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -222,6 +303,116 @@ impl fmt::Display for Height {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { fmt::Display::fmt(&self.0, f) }
 }
 
+impl ops::Add for Height {
+    type Output = Height;
+    #[inline]
+    fn add(self, rhs: Height) -> Self::Output { Height(self.0 + rhs.0) }
+}
+
+impl ops::Add<u16> for Height {
+    type Output = Height;
+    #[inline]
+    fn add(self, rhs: u16) -> Self::Output { Height(self.0 + rhs) }
+}
+
+impl ops::Add<Height> for u16 {
+    type Output = Height;
+    #[inline]
+    fn add(self, rhs: Height) -> Self::Output { Height(rhs.0 + self) }
+}
+
+impl ops::Sub for Height {
+    type Output = Height;
+    #[inline]
+    fn sub(self, rhs: Height) -> Self::Output { Height(self.0 - rhs.0) }
+}
+
+impl ops::Sub<u16> for Height {
+    type Output = Height;
+    #[inline]
+    fn sub(self, rhs: u16) -> Self::Output { Height(self.0 - rhs) }
+}
+
+impl ops::Sub<Height> for u16 {
+    type Output = Height;
+    #[inline]
+    fn sub(self, rhs: Height) -> Self::Output { Height(self - rhs.0) }
+}
+
+impl ops::Mul for Height {
+    type Output = Height;
```

Comment:
This shouldn't exists because there's no `HeaightSquared` (Or `BlockchainArea`?)